### PR TITLE
Edoardomarsili florence

### PR DESCRIPTION
**Pull request by edoardomarsili**
Test pull request

### CODE REVIEW

**Comment by edoardomarsili** [comment]
Issue #28

File: edoardomarsili-hometown.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Florence, Italy
+
+### Great Places to Eat
+
+- Lo Spumantino
+- Cantina da Verrazzano
+
```
Comment (edoardomarsili) [inline edoardomarsili-hometown.md]: Add one more place!

File: edoardomarsili-hometown.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+### Fun things to do
+
```
Comment (razor5900) [inline edoardomarsili-hometown.md]: What about music, are there good places to listen to local musicians?

Comment (edoardomarsili) [inline edoardomarsili-hometown.md]: Hey. Yup, you can go to Teatro La Pergola for classics or Teatro Comunale for a wider variety of artists. If you fancy clubs, there's few in town and outskirts :-)

**Pull request merged**